src/Symfony/Component/Ldap/Adapter/ExtLdap/EntryManager.php

```diff
@@ -110,6 +110,25 @@ public function rename(Entry $entry, $newRdn, $removeOldRdn = true)
         }
     }
 
+    /**
+     * Moves an entry on the Ldap server.
+     *
+     * @param Entry  $entry
+     * @param string $newParent
+     *
+     * @throws NotBoundException
+     * @throws LdapException
```

Comment:
we should tell "when" these will be thrown - the annotation itself is not enough